evap/evaluation/models.py

```diff
@@ -222,6 +236,18 @@ def __lt__(self, other):
     def __gt__(self, other):
         return (self.type, self.order, self.pk) > (other.type, other.order, other.pk)
 
+    @transaction.atomic()
+    def set_active_dropout(self):
+        # TODO@Felix: test this method
+        if self.type != Questionnaire.Type.DROPOUT:
+            raise ValueError("Can only set DROPOUT-type Questionnaires as active dropout questionnaire.")
+
+        Questionnaire.objects.active_dropout_questionnaire().update(is_active_dropout_questionnaire=False)
+        self.is_active_dropout_questionnaire = True
+        self.save()
+        # TODO@Felix: why does this not work [wip]
```

Comment:
I think the (unique=True, blank=True, null=True) tri-state approach forces us to have at most one instance with value `True` and all other instance with value `None`/`NULL`. The update above sets it to `False`, which would give it a unique validation failure. Guess that's one issue here